docker-wrappers/MinCostFlow/Dockerfile

```diff
@@ -0,0 +1,8 @@
+#uses ubuntu as the base
+FROM python:3.10.7
+
+WORKDIR /MinCostFlow
+
+RUN wget https://raw.githubusercontent.com/gitter-lab/min-cost-flow/ea1added536d422b5ead7ba544af4e6fe0532210/minCostFlow.py
+
+RUN pip install ortools==9.4.1874
```

Comment:
Let's switch the order of the two `RUN` commands to get better caching behavior.  We are more likely to update the version of `minCostFlow.py` frequently, and currently we will need to reinstall `ortools` every time we download a new version of the `.py` file.  If we reorder them, we can keep `ortools` cached.